workspacemanager/WorkspaceManagerPresenter.py

```diff
@@ -0,0 +1,75 @@
+from workspacemanager.command import Command
+import os.path
+
+
+#TODO askOwen is passing mtd.getObjectNames() ideal? pass mtd? Answer: Will do for time being
+#TODO When moving to actual workspaces the tests should reflect that
+#TODO implement compose workspace
+#TODO tell user when file not found,file failed to load
+
+class WorkspaceManagerPresenter(object):
+    def __init__(self,view,workspace_provider=None):
+        self._groupCount = 1
+        self._workspaceMangerView = view
+        self._workSpaceprovider = workspace_provider
+
+    def notify(self,command):
+        if command == Command.LoadWorkspace:
+            #TODO specify workspace name on load
+            #TODO what to do on fail?
+            workspace_to_load = self._workspaceMangerView.get_workspace_to_load_path()
+            base = os.path.basename(workspace_to_load)
+            ws_name = os.path.splitext(base)[0]
+            self._workSpaceprovider.Load(Filename=workspace_to_load, OutputWorkspace=ws_name)
+            self._workspaceMangerView.display_loaded_workspaces(self._workSpaceprovider.getWorkspaceNames())
+
+        if command == Command.SaveSelectedWorkspace:
```

Comment:
1. Should be an `elif` 
2. You should add a fall-through block that raises an exception if no `Command` matches! This should be a scenario in your tests.
3. Refactor this so that the "what" and "how" are better separated in the code. Notify should look more like this.

``` python
def notify(self, command):
    if command == Command.LoadWorkspace:
        load_workspace()
    elif command == Command.SaveSelectedWorkspace:
        save_selected_workspace()
#etc
```